src/services/accounts/AccountsStakingPayoutsService.ts

```diff
@@ -136,6 +136,16 @@ export class AccountsStakingPayoutsService extends AbstractService {
 
 		const sanitizedEra = era < 0 ? 0 : era;
 
+		const at: IBlockInfo = {
+			height: number.unwrap().toString(10),
+			hash,
+		};
+
+		// User friendly - we don't error if the user specified era & depth combo <= 0, instead just start at 0
+		const startEra = Math.max(0, sanitizedEra - (depth - 1));
+		const runtimeInfo = await this.api.rpc.state.getRuntimeVersion(at.hash);
+		const specName = runtimeInfo.specName.toString();
```

Comment:
Couldn't we just call `toLowerCase()` here? Since it's repeating every time we use `specName`.